### PR TITLE
react-scripts: added react-router alias

### PR DESCRIPTION
**Pull request by spacerockzero**
Required by zion-analytics
Added react-router alias without whitespace changes this time

### CODE REVIEW

File: packages/react-scripts/package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fs/react-scripts",
-  "version": "3.0.0-RC.6",
+  "version": "3.0.0-RC.7",
```
Comment (redbugz) [inline packages/react-scripts/package-lock.json]:
```suggestion
  "version": "3.0.0-RC.8",
```

File: packages/react-scripts/package.json [hunk 2]
```diff
@@ -107,7 +107,8 @@
     "i18next": "^15.1.3",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
-    "react-i18next": "^10.11.0"
+    "react-i18next": "^10.11.0",
+    "react-router": "^5.0.1"
```
Comment (redbugz) [inline packages/react-scripts/package.json]: I wonder if this is necessary since it's not a peer dep?

Comment (spacerockzero) [reply packages/react-scripts/package.json]: i got a dep not found error with my linked cra app before i added the dep.
if there's something else that works, or it's a fluke, i could remove it, but that was my experience

**Comment by redbugz** [comment]
So @joeycozza are we good to merge this? While annoying to play whack-a-mole, I don't think adding devDeps and aliases is going to cause future conflicts or problems for any case that I can think of, and the only other alternative is not allow npm linking, right?

**Comment by spacerockzero** [comment]
🎉

**Pull request merged**